### ISSUE TITLE
Not Working with v2 Data

### ISSUE THREAD

**Issue by adamtrobbins** [PrincetonUniversity/Iseult#43]
When I try to run iseult.py on '/scratch/gpfs/ar3676/1e_test' I get the below 'file format not supported' error. It was working before, I am not sure which change caused this to fail now. Perhaps it has to do with the TBB version warning. 

```
[17:06][ar3676@stellar-vis2 ~/scr/1e_test/output]$ iseult.py
/home/ar3676/.conda/envs/iseult/lib/python3.11/site-packages/numba/np/ufunc/parallel.py:371: NumbaWarning: The TBB threading layer requires TBB version 2021 update 6 or later i.e., TBB_INTERFACE_VERSION >= 12060. Found TBB_INTERFACE_VERSION = 12010. The TBB threading layer is disabled.
  warnings.warn(problem)
Traceback (most recent call last):
  File "/home/ar3676/Desktop/Iseult/iseult.py", line 69, in <module>
    runMe(cmd_args)
  File "/home/ar3676/Desktop/Iseult/src/main_app.py", line 3475, in runMe
    app = MainApp('Iseult', cmd_args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ar3676/Desktop/Iseult/src/main_app.py", line 1881, in __init__
    self.InitializeCanvas()
  File "/home/ar3676/Desktop/Iseult/src/main_app.py", line 2284, in InitializeCanvas
    self.ReDrawCanvas()
  File "/home/ar3676/Desktop/Iseult/src/main_app.py", line 3014, in ReDrawCanvas
    self.LoadAllKeys()
  File "/home/ar3676/Desktop/Iseult/src/main_app.py", line 2511, in LoadAllKeys
    self.DataDict[elm] = data_loading.load_dataset(filepath, elm, slice(None, None, self.MainParamDict['PrtlStride']))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ar3676/Desktop/Iseult/src/data_loading.py", line 391, in load_dataset
    data_version = __detect_tristan_data_version(file)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ar3676/Desktop/Iseult/src/data_loading.py", line 37, in __detect_tristan_data_version
    raise ValueError('Data file format is not supported. If this is a data file from Tristan v1 or v2 please submit a bug report.')
ValueError: Data file format is not supported. If this is a data file from Tristan v1 or v2 please submit a bug report.

'''

**Comment by bcaddy** [comment]
Are you sure it used to work on this exact dataset? Or just that it was/is working on other Tristan v2 datasets? I tried a bunch of old commits and got the same error on all of them for this dataset.

I'm pretty sure the issue is that Iseult decides if something is a Tristan v1 or v2 file based on looking if it contains specific data fields and the particle files for this dataset don't contain any common fields with other Tristan v2 particle files I've seen.  I'll post a PR with a fix shortly.

**Comment by adamtrobbins** [comment]
Thanks for checking this out. I am actually not sure it was working with this particular dataset, only that it was working with other v2 datasets. It could be related to the spectral bins since with my new runs I have been using >1 bins in x and y, perhaps this is causing some issue.

**Comment by bcaddy** [comment]
More than 1 bin in y/z is fine. I fixed this issue in PR #44, it was the issue with iseult not having keys for these files

**Comment by adamtrobbins** [comment]
All works now. Thanks!
